tsconfig.packages.build.json

```diff
@@ -12,5 +12,11 @@
     // or included in extending tsconfig files for specific parts of the project
     "types": []
   },
-  "exclude": ["./jest.config.packages.ts", "**/*.test.ts", "**/jest.config.ts"]
+  "exclude": [
+    "./jest.config.packages.ts",
+    "**/*.test.ts",
+    "**/jest.config.ts",
+    "**/*.stories.ts",
+    "**/*.stories.tsx"
```

Comment:
This ignores storybook files from the build process. I am putting it at the root tsconfig.packages.build.json level because we will also have `stories.tsx` files in both `design-system-react` and `design-system-react-native` packages